manager/src/main/java/com/redhat/service/bridge/manager/workers/resources/ProcessorWorker.java

```diff
@@ -0,0 +1,98 @@
+package com.redhat.service.bridge.manager.workers.resources;
+
+import java.util.Objects;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.infra.models.dto.ManagedResourceStatus;
+import com.redhat.service.bridge.manager.dao.ConnectorsDAO;
+import com.redhat.service.bridge.manager.dao.ProcessorDAO;
+import com.redhat.service.bridge.manager.models.ConnectorEntity;
+import com.redhat.service.bridge.manager.models.Processor;
+import com.redhat.service.bridge.manager.models.Work;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;
+import io.quarkus.vertx.ConsumeEvent;
+
+@ApplicationScoped
+public class ProcessorWorker extends AbstractWorker<Processor> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProcessorWorker.class);
+
+    @Inject
+    ProcessorDAO processorDAO;
+
+    @Inject
+    ConnectorsDAO connectorsDAO;
+
+    @Inject
+    ConnectorWorker connectorWorker;
+
+    @Override
+    protected PanacheRepositoryBase<Processor, String> getDao() {
+        return processorDAO;
+    }
+
+    // This must be equal to the Processor.class.getSimpleName()
```

Comment:
Probably better to use Fully Qualified Name instead

Reply:
OK.